File: dotnet/01-commands/fix/DemoFix/MyClassExample1.cs
using System;
namespace MyNamespace
{
    abstract public class MyClassExample1
    {
        public abstract int f();
    }

    abstract public class MyClass2
    {
        public override int f()   
        {
            return 0;
        }

        public static void Main()
        {
        }
    }
}

File: nodejs/01-commands/fix/sourcecode/MyClassExample1.js
class MyClassExample1 {
    f() {
        throw new Error('Method not implemented.');
    }
}

class MyClass2 extends MyClassExample1 {
    f() {
        return 0;
    }
}

const myClass2Instance = new MyClass2();
console.log(myClass2Instance.f());

module.exports = { MyClassExample1, MyClass2 };